src/triage/component/audition/distance_from_best.py

```diff
@@ -58,6 +58,7 @@ def _populate(self, model_group_ids, train_end_times, metrics):
                 All models should have the test_results.evaluations table populated
                 for all given model group ids, train end times, and metric/param combos
         """
+        logging.info("Polulating data to distance table")
```

Comment:
looks like we don't generally bother with this, so just a question -- do the logs just look like:

    …
    Populating data to distance table
    …

It might be nice to at least annotate such messages with the module name:

    LOG = logging.getLogger(__name__)

    ..

    LOG.info(..)

That way it should come up as something like:

    [distance_from_best] Populating data …

I don't know that that's the ideal, though. Perhaps what we really want is something like:

    [triage] [audition] …

which could also be configured.

Reply:
I agree that we should overhaul this soon.

One quick win would be to set some good formatting defaults in src/triage/cli.py's basicCOnfig (filename, linenumber, etc), but making a change to actually do file-based loggers is worth considering too.

Reply:
Can I not address the logger issue in this PR since it's a general triage logger?